src/API/OnboardingTasks.php

```diff
@@ -776,8 +776,11 @@ public function get_status() {
 	 */
 	public function get_tasks( $request ) {
 		$extended_tasks = $request->get_param( 'extended_tasks' );
-		$lists          = TaskLists::get_lists( $extended_tasks );
-		$json           = array_map(
+		TaskLists::maybe_add_extended_tasks( $extended_tasks );
```

Comment:
Looks great to me!  @louwie17 just wanted to make sure this doesn't break anything in the extended list work.

Reply:
Thanks for the ping, I think this will break the `extended_tasks` mostly because the `extended` task list doesn't exist yet when this method is called.
I think it be better if we move https://github.com/woocommerce/woocommerce-admin/blob/8400551e3f0e63dd6fff0629261edcb82706334e/src/Features/OnboardingTasks/TaskLists.php#L100-L105 to the `maybe_add_extended_tasks`, this would make more sense anyhow.

Reply:
That solution would work for me, adding the `extended` task list to the `protected static $lists` is optional (in my opinion).